main.py

```diff
@@ -1,16 +1,39 @@
+import json
 import os
 
 from flask import Flask
+from flask import request
+
+from jit.core import jit_cleaner
+from jit.utils import config
+from jit.utils.logger import jit_logger
 
 app = Flask(__name__)
 
+conf = config.parse_env()
+
 
-@app.route("/")
+@app.route("/hello")
```

Comment:
hmm, perhaps let's change this into a health check endpoint, rather than a hello world endpoint.

Reply:
got it. changing it.

Reply:
done. i directly removed the api since it is not used anywhere.